File: Data Structures & Algorithms/max-water-container/submission-1.java
class Solution {
    public int maxArea(int[] h) {
        int ans = 0;

        int l = 0, r = h.length - 1;
        while (l < r) {
            int i = Math.min(h[r], h[l]);
            int j = r - l;
            int vol = i * j;
            System.out.println(i + " " + j);
            ans = Math.max(ans, vol);
            if (h[l] < h[r]) {
                l++;
            } else {
                r--;
            }
        }
        return ans;
    }
}

File: Data Structures & Algorithms/majority-element/submission-1.cpp
class Solution {
   public:
    int majorityElement(vector<int>& nums) {
        unordered_map<int, int> mp;
        int ans = 0, max = 0;
        
        for (int n : nums) {
            mp[n]++;
            if (mp[n] > max) {
                ans = n;
                max = mp[n];
            }
        }
        return ans;
    }
};